src/auth/auth.service.ts

```diff
@@ -87,4 +89,23 @@ export class AuthService {
             access_token: this.jwtService.sign(payload)
         }
     }
+
+    async isValidJWT(userId: ObjectId): Promise<boolean> {
+        let user: User
```

Comment:
We could make a const of that when we move the logic into the try in catch. However it's better readable like that.

Reply:
No not possible as declaring a const in a try catch block leads to the const not being accessible outside the try catch block